.github/workflows/release.yaml

```diff
@@ -0,0 +1,29 @@
+on:
```

Comment:
nit (optional): could add `name: Release` here so it looks nicer on github:
![image](https://github.com/Argus-Labs/cardinal-editor/assets/51780559/ec143f22-0c2a-4195-ad94-fde4d1c5918c)

Reply:
Ah missed that one, thanks!